src/dev-server/dev-server-client/polyfills.ts

```diff
@@ -24,12 +24,12 @@ const applyObjectAssign = () => {
   if (typeof Object.assign !== 'function') {
```

Comment:
I think for our shims, it's safer to leave `var` in place. They (the shims) are designed to run in older browsers (like Safari 10, IE 11). But I think these shims will go away in a future major release where we don't support those browsers. For now, can we ignore 'no-var' here?